diff --git a/native/install.sh b/native/install.sh
index f3c3aca..e365fca 100755
--- a/native/install.sh
+++ b/native/install.sh
@@ -6,7 +6,7 @@
 
 set -eu
 
-EXTENSION_NAME="job_search_background"
+EXTENSION_NAME="job_search_background.prod"
 EXTENSION_BACKEND_PATH=$(realpath $0 | xargs dirname)
 EXTENSION_BIN=${EXTENSION_BACKEND_PATH}/run.sh
 
diff --git a/src/Background.purs b/src/Background.purs
index b73fd21..86d7396 100644
--- a/src/Background.purs
+++ b/src/Background.purs
@@ -28,7 +28,7 @@ import Web.URL as URL
 main :: Effect Unit
 main = do
   log "[bg] starting up"
-  port <- connectToNativeApplication "job_search_background"
+  port <- connectToNativeApplication "job_search_background.prod"
   -- TODO: This is redundant to send "port" to both addListener and the handler
   onNativeMessageAddListener port $ nativeMessageHandler port
   onNativeDisconnectAddListener port \p -> log $ "disconnected from native port " <> p.name <> " (" <> p.error <> ")"
